addons/core/addon/components/session-status/index.js

```diff
@@ -0,0 +1,20 @@
+import Component from '@glimmer/component';
+
+const STATUS_MAPPING = {
+  //map session status to hds text color and icon style
+  //more info - https://design-system-components-hashicorp.vercel.app/components/badge
+  active: ['success', 'check'],
+  pending: ['neutral', 'delay'],
+  canceling: ['critical', 'x'],
```

Comment:
Should cancelling be a warning color with a distinct icon from terminated?  Potentially makes it easier to scan a list with these.